contents/julia_101.md

```diff
@@ -1,40 +1,438 @@
 # Julia 101 {#sec:julia_101}
 
-We should include a nice warning box here saying:
-
 > **_NOTE:_**
 > In this chapter we will cover the basic of Julia as a programming language.
 > Please note that this is not *strictly necessary* for you to use Julia as a tool of data manipulation and data visualization.
 > Having a basic understanding of Julia will definitely make you more *effective* and *efficient* in using Julia.
 > However, if you prefer to get started straight away, you can jump to [@sec:dataframes] to learn about tabular data with DataFrames.jl.
 
+This is going to be a very brief and *not* in-depth overview of Julia language.
+If you are already familiar and confortable with other programming languages, we highly encourage you to read [Julia's documentation](https://docs.julialang.org/).
+It is a thoroughly deep dive into Julia.
+It covers all the basics and corner cases, but it can be cumbersome.
+Specially if you are a beginner into programming and haven't yet become familiar with opensource software documentation.
+
+We'll cover just the basics of Julia.
+Since we love analogies, imagine that Julia is a fancy feature-loaded car, such as a brand-new Tesla.
+We'll just explain you how to "drive the car, park it and how to navigate in traffic".
+If you want to know what all "buttons in the steering wheel and dashboard do", this is not the resource you are looking for.
+
 ## Language Syntax {#sec:syntax}
-### Functions {#sec:fun}
-### For Loop {#sec:for}
-### While Loop {#sec:while}
-### Conditional If-Else-Elseif {#sec:conditionals}
-### User-defined Types
+
+Julia is a **dynamic-typed language**.
+This means that you don't need to compile it before you run code, like you would do in C++ or FORTRAN.
+The main differences from Julia to other dynamic languages such as R and Python are the following.
+
+First, Julia, contrary to most dynamic-typed languages, **enables the user to specify type declaration**.
+You already saw some types declarations in the [*Why Julia?*](why_julia.html) section: they are those double colon `::` that suffixes the variables inside `struct`s and `function`'s calls.
```

Comment:
```suggestion
You already saw some types declarations in the [*Why Julia?*](why_julia.html) section: they are those double colon `::` that sometimes comes after variables
```
Counterexample to "`struct`s and `function`'s calls":
```
let x=1
    y::Int = 2
end
```
is valid as are many more situations